package.json

```diff
@@ -9,9 +9,8 @@
   "scripts": {
     "flowzone-preinstall": "sudo apt-get update && sudo apt-get install -y libudev-dev",
     "test": "npm run lint && mocha -r ts-node/register tests/**/*.spec.ts",
-    "prettier": "balena-lint --fix lib typings examples tests",
     "lint": "balena-lint lib typings examples",
-    "lint:fix": "balena-lint --fix lib typings examples",
+    "lint-fix": "balena-lint --fix lib typings examples tests",
```

Comment:
I standardized on this naming to match where all our repos are gradually heading